### PR TITLE
Add latest post

### PR DESCRIPTION
**Pull request by peachpellen**

### CODE REVIEW

**Comment by blundell** [comment]
@peachpellen just need to merge master , you can do it from the webinterface ^

File: posts/peachpellen/Dealing_with_Android_Fragmentation.jade [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+	###Good Emulators
+	You'll end up doing more than half of your testing on emulators. Unfortunately, the official Android emulator is so slow to launch that it makes even those of us who've used it a thousand times wonder whether or not something went wrong. [Genymotion](https://www.genymotion.com/#!/) solves that problem. It's free for basic testing, with some premium options if you need them, and it allows you to see a very reliable simulation of what you'll see on a device; including less than desirable performance, depending on your settings. (I've found it especially useful for positioning and sizing images before I'm ready to test on devices.)
```
Comment (blundell) [inline posts/peachpellen/Dealing_with_Android_Fragmentation.jade]: :+1:

**Comment by peachpellen** [comment]
Sorry Paul, there are parts of Github I'm very familiar with and other parts that I'm simply clueless about. I am hoping that last commit will fix the issue, though - my branch is up to date and I just converted tabs to spaces as it seems that was the issue... and yes; it just went green :-) Great.

**Comment by blundell** [comment]
I always get that tab to space error, so annoying

**Pull request merged**